.github/workflows/release.yml

```diff
@@ -73,7 +80,85 @@ jobs:
           registry-url: "https://registry.npmjs.org"
       - name: Publish package
         id: publish
-        uses: slsa-framework/slsa-github-generator/actions/nodejs/publish@v2.0.0
+        uses: slsa-framework/slsa-github-generator/actions/nodejs/publish@5a775b367a56d5bd118a224a811bba288150a563 # v2.0.0
+        with:
+          access: public
+          node-auth-token: ${{ secrets.NPM_TOKEN }}
+          package-name: ${{ needs.slsa.outputs.package-name }}
+          package-download-name: ${{ needs.slsa.outputs.package-download-name }}
+          package-download-sha256: ${{ needs.slsa.outputs.package-download-sha256 }}
+          provenance-name: ${{ needs.slsa.outputs.provenance-name }}
+          provenance-download-name: ${{ needs.slsa.outputs.provenance-download-name }}
+          provenance-download-sha256: ${{ needs.slsa.outputs.provenance-download-sha256 }}
+
+
+jobs:
+  build:
```

Comment:
Smells like there's some c&p chicanery going on here -- cuz doubled-up jobs blocks shouldn't be a thing, right?

Reply:
good catch, the paste was weird from the remediation tool